src/components/App.js

```diff
@@ -3,13 +3,16 @@ import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import '../App.css';
 
-function App() {
-  return (
-    <div id="app" className="App">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
Rather than overriding the **eslint** rule here you should make this a `functional component` and change it to a `stateful component` back when you get to milestone 5 where you'll require state management. 

This is the process in app development when the requirement grows, you make necessary changes.